java/vector/src/test/java/org/apache/arrow/vector/ipc/BaseFileTest.java

```diff
@@ -109,12 +114,13 @@ public void tearDown() {
   }
 
 
-  private static short [] uint1Values = new short[]{0, 255, 1, 128, 2};
-  private static char [] uint2Values = new char[]{0, Character.MAX_VALUE, 1, Short.MAX_VALUE * 2, 2};
-  private static long [] uint4Values = new long[]{0, Integer.MAX_VALUE + 1L, 1, Integer.MAX_VALUE * 2L, 2};
-  private static BigInteger[] uint8Values = new BigInteger[]{BigInteger.valueOf(0),
+  private static final short [] uint1Values = new short[]{0, 255, 1, 128, 2};
+  private static final char [] uint2Values = new char[]{0, Character.MAX_VALUE, 1, Short.MAX_VALUE * 2, 2};
+  private static final long [] uint4Values = new long[]{0, Integer.MAX_VALUE + 1L, 1, Integer.MAX_VALUE * 2L, 2};
+  private static final BigInteger[] uint8Values = new BigInteger[]{BigInteger.valueOf(0),
       BigInteger.valueOf(Long.MAX_VALUE).multiply(BigInteger.valueOf(2)), BigInteger.valueOf(2),
       BigInteger.valueOf(Long.MAX_VALUE).add(BigInteger.valueOf(1)), BigInteger.valueOf(2)};
+  private static final UUID[] uuidValues = new UUID[] {UUID.randomUUID(), UUID.randomUUID(), UUID.randomUUID()};
```

Comment:
nit: Would this guarantee 3 unique values or is it irrelevant here?

Reply:
@vibhatha if it doesn't, it's time to buy a lottery ticket :slightly_smiling_face: 

Joking aside, we could hard-code three UUID literals here?

Reply:
I think random is fine